UI/CMakeLists.txt

```diff
@@ -363,6 +363,10 @@ if(OS_WINDOWS)
     target_link_options(obs PRIVATE /LARGEADDRESSAWARE)
   endif()
 
+  if(TARGET OBS::browser-panels)
```

Comment:
Make this global for all OSes and guard behind `TARGET OBS::browser-panels` and `ENABLE_WHATSNEW`.